pkg/network/multus/annotation_test.go

```diff
@@ -102,6 +106,154 @@ var _ = Describe("Multus annotations", func() {
 			)
 		})
 	})
+
+	Context("Merge Multus annotations", func() {
```

Comment:
I think it would be preferable to test the new logic from the caller's perspective [1], as the overall scenario is clearer from that level. It would also allow us to cover all the new logic, which isn't quite possible from here.

[1] https://github.com/kubevirt/kubevirt/blob/d54a253771f366c6f7fbf191a139b4d3bdf719be/pkg/network/pod/annotations/generator_test.go#L473